CHANGELOG.md

```diff
@@ -29,6 +29,13 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+## 1.0.1a1
+
+**May 6, 2024**
+- Fixed `download_from_gcs_unpack` tempdir set now to `output_dir` previously set to default temp folder
+- Renaming misnamed test for `download_from_gcs_unpack` previously named as `test_download_gcs_not_unpack`
+- Adding missing test coverage for `test_download_gcs_not_unpack`
```

Comment:
test_download_gcs_not_unpack -> download_gcs_not_unpack

Reply:
Remove these tests as we do not provide full coverage yet. I will create a separate ticket for that.